org.moneymanagerex.MMEX.yml

```diff
@@ -0,0 +1,83 @@
+app-id: org.moneymanagerex.MMEX
+runtime: org.gnome.Platform
```

Comment:
If you need to build webkit2 anyway then you switch back to `org.freedesktop.Platform` and `org.freedesktop.Sdk` version 22.08 if there are no other gnome specific deps.

Reply:
It would appear that more dependencies would have to be build if switching to `org.freedesktop.Platform`:

```
-- Could NOT find WPE (missing: WPE_LIBRARY WPE_INCLUDE_DIR) (Required is at least version "1.3.0")
CMake Error at Source/cmake/OptionsGTK.cmake:292 (message):
  libwpe is required for USE_WPE_RENDERER
Call Stack (most recent call first):
  Source/cmake/WebKitCommon.cmake:227 (include)
  CMakeLists.txt:25 (include)
```

Reply:
Ok, then keep it as is., Thx for checking.

Reply:
If I disable enough features, I can get this to build with `org.freedesktop.Platform`:

```diff
diff --git a/org.moneymanagerex.MMEX.yml b/org.moneymanagerex.MMEX.yml
index 33bfdde..fdfbdb2 100644
--- a/org.moneymanagerex.MMEX.yml
+++ b/org.moneymanagerex.MMEX.yml
@@ -1,7 +1,7 @@
 app-id: org.moneymanagerex.MMEX
-runtime: org.gnome.Platform
-runtime-version: '44'
-sdk: org.gnome.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
 command: mmex
 rename-desktop-file: mmex.desktop
 rename-icon: mmex
@@ -42,6 +42,10 @@ modules:
       - -DENABLE_GAMEPAD=OFF
       - -DENABLE_WEBDRIVER=OFF
       - -DUSE_SOUP2=ON
+      - -DUSE_WPE_RENDERER=OFF
+      - -DUSE_LIBSECRET=OFF
+      - -DENABLE_SPELLCHECK=OFF
+      - -DUSE_WOFF2=OFF
     modules:
       - name: libsoup-2.4
         buildsystem: meson
```

That being said, I'm not comfortable switching all of these features off without confirmation with upstream that they're not being used.

Reply:
> If you need to build webkit2 anyway then you may switch back to `org.freedesktop.Platform` and `org.freedesktop.Sdk` version 22.08 if there are no other gnome specific deps.

using GNOME 44 is NOT a problem.